Add reference to supported languages (#5979)
testcontainers/testcontainers-java: docs/supported_languages.md

> # Supported Languages

Reviewer: ```suggestion
# Other Supported Languages
```

> You can find Testcontainers implemented in other languages:

Reviewer: ```suggestion
Besides *Testcontainers for Java,* you can find Testcontainers implementations for many other languages.
Please note that not all features that are available in *Testcontainers for Java* might be available in the implementations for other languages yet.
```

* [Go](https://golang.testcontainers.org/)
* [.Net](https://github.com/testcontainers/testcontainers-dotnet)

> * [Python](https://testcontainers-python.readthedocs.io/)

Reviewer: ```suggestion
* [Python](https://testcontainers-python.readthedocs.io/)
* [JavaScript/NodeJS](https://github.com/testcontainers/testcontainers-node)
```
